tests/docker-compose.yml

```diff
@@ -67,34 +67,41 @@ services:
       - ${LZMWEBPORT}:80
     depends_on:
       - lizmap
+  # Qgis RCP backend
+  map-rpc:
+    container_name: "lizmap${LZMBRANCH}_test_qgis_rpc"
+    image: 3liz/qjazz:${LZMQGSRVVERSION}
+    command: ["qjazz-rpc", "serve", "-C", "/srv/etc/qgis-server.toml"]
+    environment:
+      CONF_LOGGING__LEVEL: debug
+      CONF_WORKER__NUM_PROCESSES: 4
```

Comment:
```suggestion
      CONF_LOGGING_LEVEL: debug
      CONF_WORKER_NUM_PROCESSES: 4
```
Is-it expected ?

Reply:
Do not change this, this is a standard mapping from env variables for structured configuration schema, double underscore '__' is section separator. This is automaticalled parsed and used by framework like Pydantic in Python or Rust configuration parser.